src/main/resources/properties/environment/prod/project.properties

```diff
@@ -0,0 +1,2 @@
+system.allure.link.issue.pattern=https://myproject.jira.com/issues/{}
+system.allure.link.tms.pattern=hthttps://myproject.testrail.io/index.php?/cases/view/{}
```

Comment:
does it make sense to add a comment these links are stubs and added to demo patterns?